versions.yaml

```diff
@@ -271,6 +271,12 @@ externals:
     # create a "latest" entry and use that for the GitHub actions tests.
     latest: "v1.29"
 
+  runc:
+    description: "CLI tool for spawning and running containers"
+    url: "https://github.com/opencontainers/runc"
+    version: "v1.2.3"
+    latest: "v1.2"
+
```

Comment:
Can we try to avoid having the runc dependency pinned here and always go for the latest one?

As this is only for testing, and not something we're shipping / depending on for our builds, I think we may end up with this one being not updated quite easily.

Reply:
Hi @fidencio 

I had just add the "latest" basin version here, which means it would use the latest version based on the basic version, and it should take a long time to upgrade the basic version.